gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -838,7 +832,13 @@ def AcceptLevelUp():
 			GemRB.SetPlayerStat (pc, IE_LEVEL2, GemRB.GetPlayerStat (pc, IE_LEVEL2)+NumOfSecoLevUp)
 	
 	LUSkillsSelection.SkillsSave (pc)
-	UpdateRecordsWindow ()
+
+	GUICommonWindows.OptionsWindow.SetVisible (True)
```

Comment:
I'm trying to figure out why this is required, but even if it is I believe `SetGameGUIHidden(False)` will do this all in one line

Reply:
NewLife.py hides all the windows because it loads the stat selection window for the Nameless One (regardless of which character is levelling). That's a fullscreen window like the start screen or bio screen.

Although, that also needs maintenance, so I can add that to the todo list

Reply:
im not sure I understand. why does that translate into needing to hide the GUI? you said it is a fullscreen window so who cares what else is underneath it.

Reply:
You're right, it works fine without it, so I will remove it all.